viewing_party/party.py

```diff
@@ -1,23 +1,180 @@
 # ------------- WAVE 1 --------------------
-
+#adding a comment in the code to commit :)
 def create_movie(title, genre, rating):
-    pass
+    
+    user_data = {
+        "watched": [],
+        "watchlist": []
+        }
+
+    new_movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if None in new_movie.values():
+        return None
+
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+    
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    watchlist_length = len(user_data['watchlist'])
+
+    """compare the title argument to value of title in
+    each movie dictionary in the user's watchlist
+    if the titles are the same then append the movie dictionary
+    to the user's watched list and delete the dictionary
+    from the user's watchlist"""
+
+    for i in range(watchlist_length):
+        if user_data['watchlist'][i]['title'] == title:  
+            user_data['watched'].append(user_data['watchlist'][i])
+            del user_data['watchlist'][i]

```

Comment:
Nice -- I haven't personally used the `del` keyword before! Between `.remove()` and `.pop()` and `del`, there's so many ways to remove items from a list (and for fun, I went down [this stack overflow thread](https://stackoverflow.com/questions/11520492/difference-between-del-remove-and-pop-on-lists) for the difference!!)